src/main/features/claim/views/eligibility/not-eligible.njk

```diff
@@ -21,58 +21,57 @@
 
       {% if reason === NotEligibleReason.CLAIM_VALUE_NOT_KNOWN %}
         <p>{{ t('You need to know the claim amount to use this service.') }}</p>
-        <p>{{ t('If you can’t calculate the claim amount, for example you”re claiming for an injury or accident, ' +
-        ' use the <a target="_blank" href="{{ n1FormUrl }}">form N1 </a> '+ newWindow() + '.' ,
+        <p>{{ t('If you can’t calculate the claim amount, for example because you’re claiming for an injury or accident, ' +
+        ' use the <a target="_blank" href="{{ n1FormUrl }}">N1 paper form</a> '+ newWindow() + '.' ,
           {  n1FormUrl : n1FormUrl }) | safe }}</p>
         {{ paperFormAddress() }}
         </p>
       {% elseif reason === NotEligibleReason.CLAIM_ON_BEHALF %}
         <p>{{ t('This service is currently for claimants representing themselves.') }}</p>
-        <p>{{ t('If you’re a legal representative <a target="_blank" href="{{ MCOL }}">use the Money Claim Online (MCOL) service.</a>',
-          { MCOL: 'https://www.moneyclaim.gov.uk/web/mcol/welcome' }) | safe }}
+        <p>{{ t('If you’re a legal representative <a target="_blank" href="{{ MCOL }}">use the Money Claim Online (MCOL) service</a> or ' +
+            'use <a target="_blank" href="{{ n1FormUrl }}">form N1</a> '+ newWindow() + ' to make your claim.',
+          { MCOL: 'https://www.moneyclaim.gov.uk/web/mcol/welcome', n1FormUrl : n1FormUrl }) | safe }}
```

Comment:
this should have been: legacyServiceUrl i.e. its in a var

Reply:
this entire page needs a tech debt refactoring storing to remove the sprintfs and make sure all the links are in variables